### PR TITLE
Seo phase ii

### PR DESCRIPTION
**Pull request by Opoku-Agyemang**

### CODE REVIEW

File: src/_main_/settings.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 SECRET_KEY =  os.environ.get("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (Opoku-Agyemang) [inline src/_main_/settings.py:50]: ignore.  will remove before merge

File: src/_main_/settings.py [hunk 2]
```diff
@@ -61,6 +61,7 @@
     'dev-api-env.eba-nfqpwkju.us-east-2.elasticbeanstalk.com',
     'massenergize-canary-api.us-east-2.elasticbeanstalk.com',
     '.massenergize.test',
+    '*'
```
Comment (Opoku-Agyemang) [inline src/_main_/settings.py:64]: ignore. will remove before merge

File: src/api/handlers/community.py [hunk 3]
```diff
@@ -221,3 +221,21 @@ def super_admin_list(self, request):
     if err:
       return MassenergizeResponse(error=str(err), status=err.status)
     return MassenergizeResponse(data=communities)
+
+
+  # @admins_only
```
Comment (Opoku-Agyemang) [inline src/api/handlers/community.py:226]: ignore. will uncomment this before merge

File: src/api/handlers/community.py [hunk 4]
```diff
@@ -221,3 +221,21 @@ def super_admin_list(self, request):
...
+  def add_custom_website(self, request):
+    context: Context  = request.context
+    args: dict = context.args
+    # verify the body of the incoming request
+    self.validator.expect("website", str, is_required=True)
+    self.validator.expect("subdomain", str)
+    self.validator.expect("community_id", str)
```
Comment (BradHN1) [inline src/api/handlers/community.py:233]: this should be int ?

**Review by BradHN1 — APPROVED** [review]
Looks good.  Ready to deploy and test on dev

File: src/_main_/settings.py (hunk 1 above)
Comment (BradHN1) [reply src/_main_/settings.py:50]: I removed

File: src/_main_/settings.py (hunk 2 above)
Comment (BradHN1) [reply src/_main_/settings.py:64]: I removed

File: src/api/handlers/community.py (hunk 3 above)
Comment (BradHN1) [reply src/api/handlers/community.py:226]: Done

File: src/api/store/utils.py [hunk 5]
```diff
@@ -32,19 +33,35 @@ def get_user(user_id, email=None):
...
+  elif subdomain and subdomain != 'home':
+    community = Community.objects.select_related("logo", "favicon", "goal").filter(subdomain__iexact=subdomain).first()
+
+  elif referrer_website:
+    # since we dont have a subdomain or community_id, we will fall back to the referrer site
+    website = strip_website(referrer_website)
+    community_website = CustomCommunityWebsiteDomain.objects.select_related("community").filter(website__iexact=website).first()
+    if community_website:
+      community = community_website.community
+    elif website.endswith("massenergize.org") or website.endswith("massenergize.test:3000"):
```
[the remark this answers is not in the document]
Comment (BradHN1) [reply src/api/store/utils.py:53]: done

**Pull request merged**